packages/web-twig/src/Resources/components/FileUploader/FileUploaderAttachment.twig

```diff
@@ -0,0 +1,20 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _removeText = props.removeText | default('Remove') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'FileUploaderAttachment' -%}
+{%- set _removeClassName = _rootClassName ~ '__remove' -%}
+
+{# Miscellaneous #}
+{%- set _classNames = [ _rootClassName, _class ] -%}
+
+<li {{ mainProps(props) }} {{ classProp(_classNames) }}>
+    <Icon name="file" isReusable={ false } />
```

Comment:
@dlouhak In Twig you must pass a boolean as a boolean, not a string, e. g. `isReusable={ false }` and `isReusable={{ false }}` are fine. But not this one `isReusable="{{ false }}"`.

Reply:
Thank you for reminding. Because we use `boolprop` to ensure prop type in the DS component's API, string value works, and that's the reason why I was confused. 🙈

Reply:
As agreed today, we will open selected icon names so they can be changed by developers. For `FileUploader`, it will be the main icon in `FileAttachment` and `FileUploaderInput`.